core/src/main/java/org/testcontainers/containers/ContainerState.java

```diff
@@ -43,8 +43,18 @@
      * Get the IP address that this container may be reached on (may not be the local machine).
      *
      * @return an IP address
+     * @see #getHost()
      */
     default String getContainerIpAddress() {
+        return getHost();
+    }
+
+    /**
+     * Get the host that this container may be reached on (may not be the local machine).
+     *
+     * @return a host
```

Comment:
```suggestion
     * @return host IP address
```

Is this correct?

Reply:
See https://github.com/testcontainers/testcontainers-java/pull/2742#discussion_r425755455